### PR TITLE
Some integration-related fixes

### PR DESCRIPTION
**Pull request by zuluecho9**
A few things that came to attention that were easier fixes: 

* Removed the 'Intro to open source integrations built with Telemetry SDKs' doc, due to it not being that relevant anymore and the integrations in that docs section not all being built w/ the SDKs. 
* Finessing links to that in various places and improving verbiage generally.

### CODE REVIEW

File: src/content/docs/data-apis/custom-data/intro-custom-data.mdx [hunk 1]
```diff
@@ -9,12 +9,12 @@ To get the most out of New Relic, you may need or want to report custom data to
...
 * Report data from your operating system or network. If you're using our infrastructure agent, you might want to use our [Flex integration](/docs/integrations/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration), which lets you use simple config files to report data.  
-* Build a [New Relic One application](/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application). You can build an app that lives on our platform, that can be shared with your team or the public, and that uses your own JavaScript UI functionality. You can analyze New Relic data, or bring in whatever data you want. 
+* Build a [New Relic One application](/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application) or customize an existing one from [the app catalog](https://opensource.newrelic.com/nerdpacks). You can build an app that lives on our platform, that can be shared with your team or the public, and that uses your own JavaScript UI functionality. You can analyze New Relic data, or bring in whatever data you want.
```
Comment (rhetoric101) [inline src/content/docs/data-apis/custom-data/intro-custom-data.mdx:20]: @zuluecho9, this final item in the list is really about a custom program with New Relic--not really about bringing in data like an integration, right? I wonder if this should be in the list here.

File: src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx [hunk 2]
```diff
@@ -25,6 +25,10 @@ redirects:
   - /docs/platform/platform-terminology
   - /docs/more-integrations/open-source-telemetry-integrations
   - /docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic-integrations
+  - /docs/more-integrations/open-source-telemetry-integrations/get-started/introduction-new-relics-open-source-telemetry-integrations
```
Comment (rhetoric101) [inline src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx:28]: @zuluecho9, I think this document is on the short-list for removal, but we can still grab these redirects if this doc is  ever deleted.

**Review by rhetoric101 — APPROVED** [review]
@zuluecho9, thanks for tackling all of these loose ends! I just added a few suggestions, but you can merge when you are ready.

File: src/content/docs/data-apis/custom-data/intro-custom-data.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/data-apis/custom-data/intro-custom-data.mdx:20]: they actually can use NR1 apps to bring in any sort of data from any arbitrary source (e.g., it's like building any website/app where you can bring in any data you want) so I consider it a type of solution for bringing in any custom data.

**Pull request merged**